client/lib/push-notifications/service-worker.js

```diff
@@ -0,0 +1,20 @@
+/**
+ *  This file is served as-is as /service-worker.js
+ **/
+
+/* eslint-disable */
+'use strict';
+/* eslint-enable */
+
+self.addEventListener( 'push', function( event ) {
+	var notification = event.data.json();
+
+	event.waitUntil(
+		// notifications look best with just a title
```

Comment:
Is there any reason this comment need to be in the service worker script that gets sent to the browser?